runtime/src/main/java/org/corfudb/protocols/wireprotocol/MultipleReadRequest.java

```diff
@@ -16,24 +16,24 @@
 @Data
 @AllArgsConstructor
 public class MultipleReadRequest implements ICorfuPayload<MultipleReadRequest> {
-    @Getter
+
     final List<Long> addresses;
 
+    final boolean cacheable;
```

Comment:
Can you rename it to cacheableOnServer or something which specifies the server.
It can be confused with the client side cache.

Reply:
Done.